<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSpatialObjectsWidget</class>
 <widget class="qSlicerWidget" name="qSlicerSpatialObjectsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="VisibilityCheckBox">
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
     width: 21px;
     height: 21px;
 }

 QCheckBox::indicator:checked {
     image: url(:/Icons/Medium/SlicerVisible.png);
 }
QCheckBox::indicator:unchecked {
     image: url(:/Icons/Medium/SlicerInvisible.png);
 }</string>
     </property>
     <property name="text">
      <string>Visibility</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="OpacityLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="ctkSliderWidget" name="OpacitySlider">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="GroupBox_ColorBy">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Color By:</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="GroupBox_ColorByScalars">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Scalar Colormap</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="qMRMLColorTableComboBox" name="ColorByScalarsColorTableComboBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="renameEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="Layout_ScalarPropertySelect">
           <item>
            <widget class="QRadioButton" name="ColorByScalarRadioButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Of Scalar Value</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="ctkVTKDataSetArrayComboBox" name="ColorByScalarComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="ctkRangeWidget" name="ScalarRangeWidget"/>
         </item>
         <item>
          <widget class="qMRMLRangeWidget" name="ColorByScalarInvariantDisplayRange">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ScalarRangeLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Set range:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Layout_SolidColorSelector">
        <item>
         <widget class="QCheckBox" name="ColorBySolidColorCheckBox">
          <property name="text">
           <string>Solid color</string>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="ctkColorPickerButton" name="ColorBySolidColorPicker"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="MaterialPropertyGroupBox">
        <property name="title">
         <string>&amp;Material Properties</string>
        </property>
        <property name="collapsed">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="ctkMaterialPropertyWidget" name="MaterialPropertyWidget"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="ctkCollapsibleGroupBox" name="IntersectionCollapsibleGroupBox">
        <property name="title">
         <string>Slice Intersection</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Slice interection:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="SliceIntersectionCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Slice intersection thickness:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="qMRMLSpinBox" name="SliceIntersectionThicknessSpinBox">
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
           <property name="quantity">
            <string>length</string>
           </property>
           <property name="unitAwareProperties">
            <set>qMRMLSpinBox::MaximumValue|qMRMLSpinBox::Precision|qMRMLSpinBox::Prefix|qMRMLSpinBox::Suffix</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLRangeWidget</class>
   <extends>ctkRangeWidget</extends>
   <header>qMRMLRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLSpinBox</class>
   <extends>ctkDoubleSpinBox</extends>
   <header>qMRMLSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkVTKDataSetArrayComboBox</class>
   <extends>QComboBox</extends>
   <header>ctkVTKDataSetArrayComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkColorPickerButton</class>
   <extends>QPushButton</extends>
   <header>ctkColorPickerButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkMaterialPropertyWidget</class>
   <extends>QWidget</extends>
   <header>ctkMaterialPropertyWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkRangeWidget</class>
   <extends>QWidget</extends>
   <header>ctkRangeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>VisibilityCheckBox</tabstop>
  <tabstop>ColorByScalarRadioButton</tabstop>
  <tabstop>ColorByScalarComboBox</tabstop>
  <tabstop>ColorBySolidColorCheckBox</tabstop>
  <tabstop>ColorBySolidColorPicker</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerSpatialObjectsWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ColorByScalarsColorTableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VisibilityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>OpacitySlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
